data/src/main/java/com/example/data/backendapi/dto/Customer.java

```diff
@@ -0,0 +1,63 @@
+package com.example.data.backendapi.dto;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+
+import java.util.Objects;
+
+/**
+ * CLient's objects with personal information
+ */
+@Entity
```

Comment:
I do not think it is DTO object. That is fair Entity so prefer to move it into the appropriated package